webapp/src/main/java/com/box/l10n/mojito/rest/textunit/TextUnitWS.java

```diff
@@ -87,6 +90,10 @@ public class TextUnitWS {
 
   @Autowired AssetRepository assetRepository;
 
+  @Autowired SlackWarningMessageSender slackWarningMessageSender;
```

Comment:
You can mark this as optional if you update the component to only be created in certain circumstances, just be sure to update any code that interacts with it to handle it possibly being null